### ISSUE TITLE
AttributeError on start

### ISSUE THREAD

**Issue by r-remus** [Breakend/experiment-impact-tracker#40]
When using experiment-impact-tracker 0.1.8 under Mac OS X 10.16.7 and Python 3.8.5 in my own experiment like this

```python
from experiment_impact_tracker.compute_tracker import ImpactTracker
import tempfile
...
tracker = ImpactTracker(tempfile.mkdtemp())
tracker.launch_impact_monitor()
...
```

I get the following error: `AttributeError: Can't pickle local object 'processify.<locals>.process_func'`

Full log: [tracker.log](https://github.com/Breakend/experiment-impact-tracker/files/5790426/tracker.log)

**Comment by jieru-hu** [comment]
hi @r-remus. Thank you for the report!
This should have been fixed with #41 on master and will be released with the next version.
In the mean time, feel free to install from master and try again
```
 pip install git+https://github.com/Breakend/experiment-impact-tracker.git@master
```

**Comment by r-remus** [comment]
I verified your fix works for me 👍 
Great, thank you so mich!
